### PR TITLE
Role id to distinguish params across multiple roles of the same  type

### PR DESCRIPTION
**Pull request by rafaelfolco**

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -82,6 +82,9 @@
                     "type": "string",
                     "enum": ["client", "server", "all"]
                 },
+                "id": {
+                    "type": "string"
+                },
```
Comment (k-rister) [inline schema.json:87]: should we add a regex rule that ensures that an integer is provided?

Comment (rafaelfolco) [reply schema.json:87]: This is part of validation I assume. So IMHO, it is better to address this in the validation PR (WIP)

Comment (k-rister) [reply schema.json:87]: It is actually a schema property and we do similar things in many existing schemas.  For example:

https://github.com/perftool-incubator/rickshaw/blob/master/schema/benchmark.json#L12

Comment (rafaelfolco) [reply schema.json:87]: > It is actually a schema property and we do similar things in many existing schemas. For example:
> 
> https://github.com/perftool-incubator/rickshaw/blob/master/schema/benchmark.json#L12

something like...
"pattern": "^[0-9]+$"

**Review by k-rister — APPROVED** [review]

**Pull request merged**